core/extension/gdextension_interface.cpp

```diff
@@ -1415,7 +1415,7 @@ static void gdextension_editor_help_load_xml_from_utf8_chars(const char *p_data)
 #endif
 }
 
-#define REGISTER_INTERFACE_FUNC(m_name) GDExtension::register_interface_function(#m_name, (GDExtensionInterfaceFunctionPtr)&gdextension_##m_name)
+#define REGISTER_INTERFACE_FUNC(m_name) GDExtension::register_interface_function(#m_name, (GDExtensionInterfaceFunctionPtr) & gdextension_##m_name)
```

Comment:
This change is technically "wrong" as per our code style for references, but it seems whatever fix they made in clang-format 17 for the handling of math operators in macros has a bit of an issue with reference/pointer operators.

Overall it shouldn't be a big issue, and maybe future clang-format versions will fix it back.

Reply:
Now yes :) https://github.com/llvm/llvm-project/issues/83400